R/DESCRIPTION

```diff
@@ -12,8 +12,10 @@ Description: Implements a procedure for forecasting time series data based on
     that have strong seasonal effects and several seasons of historical data.
     Prophet is robust to missing data and shifts in the trend, and typically
     handles outliers well.
+URL: https://github.com/facebook/prophet
+BugReports: https://github.com/facebook/prophet/issues
 Depends:
-    R (>= 3.2.3),
+    R (>= 3.4.0),
```

Comment:
Out of interest, why was this bump needed?

Reply:
rstantools bumped it up. Seems rstan started requiring it in 2018 for C++14 https://github.com/stan-dev/rstan/commit/cd9257016132685858b01e7699d487b97cd25df8

Reply:
Interesting, thanks!

Reply:
You’re welcome! :D